### PR TITLE
minor language tweaks to switch devops to declarative, and semantic l…

### PR DESCRIPTION
**Pull request by mgreenegit**
This isn't a significant update.  I would like to change the language to say cloud computing allows declarative templates as opposed to associating devops with a tools implementation.  As I was proofing the remainder I just refactored using semantic line feed.

### CODE REVIEW

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[dsc/DscForEngineers.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-2/dsc/DscForEngineers.md) | :warning:Warning | | [Details](#user-content-dsc%2FDscForEngineers.md)

<a name="dsc%2FDscForEngineers.md"></a>
### [dsc/DscForEngineers.md](https://github.com/mgreenegit/PowerShell-Docs/blob/mgreenegit-patch-2/dsc/DscForEngineers.md)
 - **[Warning]** Ignore metadata: ms.locale. They are generated by open publish.
 - **[Warning]** Author eslesar;mgreenegit cannot be recognized in GitHub.
 - **[Warning]** Can&#39;t find output file for bread/toc.json

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C13%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201710131434397727-1759%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=y04oRjQAXn7awJWA0Aw4P0eAubnGdhHeAite3PFCj6k%3D&st=2017-10-13T15%3A00%3A44Z&se=2017-11-13T15%3A05%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: dsc/DscForEngineers.md [hunk 1]
```diff
@@ -1,55 +1,70 @@
 ---
-ms.date:  2017-06-12
-author:  eslesar
+ms.date:  2017-10-13
+author:  eslesar;mgreenegit
```
Comment (zjalexander) [inline dsc/DscForEngineers.md]: this is causing a build error, please remove your name

File: dsc/DscForEngineers.md [hunk 2]
```diff
@@ -1,55 +1,70 @@
...
 
 ## Benefits of Desired State Configuration
 
 DSC exists to:
+
 - Decrease the complexity of scripting in Windows
 - Increase the speed of iteration
 
-The concept of "continuous deployment" is becoming more important. 
+The concept of "continuous deployment" is becoming more important.
```
Comment (zjalexander) [inline dsc/DscForEngineers.md]: I'm aware i'm the original author of this line, but in hindsight it is not very useful and I think we can just remove it altogether :)

File: dsc/DscForEngineers.md [hunk 3]
```diff
@@ -1,55 +1,70 @@
...
-DSC is a platform that provides declarative, autonomous and idempotent (repeatable) deployment, configuration and conformance.
-The DSC platform enables you to ensure that the components of your data center have the correct configuration, which avoids errors and prevents costly deployment failures.
-By treating DSC configurations as part of application code, DSC enables continuous deployment.
-The DSC configuration should be updated as a part of the application, ensuring that the knowledge needed to deploy the application is always up-to-date and ready to be used.
-
+By getting new features through development into operation as smoothly and reliably as possible,
+you reduce time-to-value of new business logic.
+
+The move towards cloud computing
+implies a deployment solution that utilizes a "declarative" template model,
```
Comment (zjalexander) [inline dsc/DscForEngineers.md]: these two lines suggest a lot of complexity that doesn't get addressed. I think you can remove everything before "A declarative template model" and change the following sentences a bit so it defines declarative models and why they are good. Then talk about how that helps the cloud second? I think that leads to a better flow.

**Pull request merged**